### PR TITLE
guidelines for OSS users willing to contribute to the documentation

### PR DESCRIPTION
**Pull request by bcollard**
# Description

Depends on https://github.com/solo-io/hugo-theme-soloio/pull/54

- add info on how to start the local Hugo server for OSS users
- change the security scan report templates to not render when there is this new Hugo site param:  `noSecurityScan`

Fix issue https://github.com/solo-io/gloo/issues/6090
BOT NOTES: 
resolves https://github.com/solo-io/gloo/issues/6090

### CODE REVIEW

File: docs/content/contributing/documentation/editing_locally.md [hunk 1]
```diff
@@ -60,12 +60,28 @@ In the previous sections you installed the necessary tools and downloaded the co
 Navigate to the `docs` folder from the Gloo Edge repo root and run `make` to start up the site.
 
 ```bash
-SKIP_CHANGELOG_GENERATION=true make serve-site -b
+cd docs
+export GITHUB_TOKEN=foo
```
Comment (kdorosh) [inline docs/content/contributing/documentation/editing_locally.md:64]: explain why?

Comment (bcollard) [reply docs/content/contributing/documentation/editing_locally.md:64]: without any token, I got the following error:

```
$> SKIP_CHANGELOG_GENERATION=true make serve-site -b

rm -fr ./site ./resources ./site-latest ./site-versioned ./public ./ci ./gloo-temp ./themes
cd .. && GO111MODULE=on go run hack/find_latest_enterprise_version.go
if [ ! -d themes/hugo-theme-soloio ]; then git clone https://github.com/solo-io/hugo-theme-soloio themes/hugo-theme-soloio; fi
Cloning into 'themes/hugo-theme-soloio'...
remote: Enumerating objects: 4018, done.
remote: Counting objects: 100% (4018/4018), done.
remote: Compressing objects: 100% (1581/1581), done.
remote: Total 4018 (delta 2382), reused 3902 (delta 2306), pack-reused 0
Receiving objects: 100% (4018/4018), 12.31 MiB | 1.95 MiB/s, done.
Resolving deltas: 100% (2382/2382), done.
git -C themes/hugo-theme-soloio checkout v0.0.16
Note: switching to 'v0.0.16'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at 2495375 Merge pull request #53 from solo-io/fix-ul-and-code-copy
GO111MODULE=on go run cmd/generate_docs.go gen-releases -r gloo > opensource.out
Error: Could not find GITHUB_TOKEN in environment.
Usage:
   gen-releases [flags]

Flags:
  -h, --help   help for gen-releases

Global Flags:
  -r, --TargetRepo string   specify one of 'gloo' or 'glooe' (default "gloo")

2022/03/15 11:03:04 unable to run: Could not find GITHUB_TOKEN in environment.
exit status 1
make: *** [site-common] Error 1
```

I didn't test with a GITHUB_TOKEN generated for a GH account **not** part of the solo-io org.
Should we ask users to generate a GH token anyway?

**Review by artberger — COMMENTED** [review]
Thanks for adding this! I am not sure about the GitHub token

**Review by artberger — APPROVED** [review]

**Pull request merged**